### ISSUE TITLE
Figure modal a11y improvements

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#4239]
from #4194 

- Sample 4, item 21. link to enlarge the figure is just 'figure 1' this needs to be more descriptive.
- Sample 4, item 22. having enlarged the figure, the modal is not navigable - no way to dismiss it.
- Sample 4, item 32. similar issues with Table 1 as with Figure 1 above. Cannot navigate to the close button, and end up outside the modal, but without it being dismissed.

**Comment by joemull** [comment]
Refinement: We need to consider this primarily for the clean theme. The bug was discovered when testing on OLH theme, but clean theme should be prioritized. The JS utility we use for this feature is the same on both, though the HTML and CSS would of course be theme-specific.

We should consider whether there is an accessible bootstrap-provided utility for this that better integrates with clean HTML and CSS than the lightbox utility that we are currently using.

An even simpler alternative is the standard HTML element **popover** which should be considered if it is the simplest solution.

**Comment by StephDriver** [comment]
Clean theme modal buttons are being updated in #4802, so this is blocked until that merges.

**Comment by StephDriver** [comment]
the equivalent work for enlarged tables was done in #4901, so this is blocked until that is merged and it can be done on top of that for consistency.

**Comment by StephDriver** [comment]
Investigating:
- Sample 4, item 21. link to enlarge the figure is just 'figure 1' this needs to be more descriptive.
✅  this now works, it is read aloud as "Enlarge Figure 1"

- Sample 4, item 22. having enlarged the figure, the modal is not navigable - no way to dismiss it.
✅  this now works, navigation to the close button is possible.  
:x: however, having closed the user is returned to the top of the page, and not to their previous location.

- Sample 4, item 32. similar issues with Table 1 as with Figure 1 above. Cannot navigate to the close button, and end up outside the modal, but without it being dismissed.
✅ table issues were handled in #4008
